x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/Literal.java

```diff
@@ -193,6 +193,10 @@ public static Literal keyword(Source source, String literal) {
         return new Literal(source, BytesRefs.toBytesRef(literal), KEYWORD);
     }
 
+    public static Literal keyword(Source source, Collection<String> literal) {
```

Comment:
ℹ️ Make it easier to instantiate multivalued strings literals, especially in tests